lib/tasks/taxondb.rake

```diff
@@ -0,0 +1,22 @@
+require 'open3'
+require 'csv'
+task load_taxon_db: :environment do
+  t = TaxonDescription.new(version: 1)
```

Comment:
use create  instead of new/save abstract out version and descriptions as arguments.